packages/devops/scripts/ci/validateBranchName.sh

```diff
@@ -1,11 +1,43 @@
 #!/bin/bash
 
-reserved_endings=(api config export mobile admin www)
+DIR=`dirname "$0"`
+. ${DIR}/utils.sh
 
-for reserved_ending in ${reserved_endings[@]}
+MAX_LENGTH=56
+RESERVED_CHARS=('/' '\' '.' '&' '?')
```

Comment:
@rohan-bes Some of those characters (eg `?`) are not disallowed from git in the first place, so a branch name wouldn't include them. do you think it's worth removing them from this list?

I'm fine either way

Reply:
Yeah I'm fine either way too 👍

Reply:
I'll leave it as is then, to match our doco